src/components/layout/navbar/WSNotificationDropdown.tsx

```diff
@@ -54,7 +54,7 @@ const WSNotificationDropdown = () => {
     };
 
     return () => {
-      ws.close();
+      ws.close = () => console.log('Connection closed 😟');
```

Comment:
This doesn't close the connection, rather changes `ws.close()` to console.log

Reply:
Fixed ✔